src/data/builder-tools.js

```diff
@@ -1034,6 +1034,13 @@ export const Showcases = [
     getstarted: "https://docs.gomaestro.org/",
     tags: ["http", "hosted"]
   },
+  {
+    title: "potential-robot",
+    description: "A JavaScript API for test-driven development with Helios.",
+    preview: require("./builder-tools/potential-robot.png"),
+    website: "https://github.com/aleeusgr/potential-robot",
+    tags: ["javascript", "typescript", "testing"]
```

Comment:
```suggestion
    getstarted: null,
    tags: ["javascript", "typescript", "testing"]
```
oops, we both forgot this required field (the Portal will not build unless this is present for every entry)